src/components/ContentHighlights/HighlightStepper/HighlightStepperTitleInput.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { useContextSelector } from 'use-context-selector';
+import { Form } from '@edx/paragon';
+
+import { ContentHighlightsContext } from '../ContentHighlightsContext';
+import { HIGHLIGHT_TITLE_MAX_LENGTH } from '../data/constants';
+import { useContentHighlightsContext } from '../data/hooks';
+
+const HighlightStepperTitleInput = () => {
+  const { setHighlightTitle } = useContentHighlightsContext();
+  const highlightTitle = useContextSelector(ContentHighlightsContext, v => v[0].stepperModal.highlightTitle);
+  const [titleLength, setTitleLength] = useState(highlightTitle?.length || 0);
+  const [isInvalid, setIsInvalid] = useState(false);
+
+  const handleChange = (e) => {
+    if (e.target.value.length > 60) {
+      setIsInvalid(true);
+      setHighlightTitle({
```

Comment:
[note] A test will be added in future PR for this coverage.

Reply:
Leaving open as a reminder to complete